astropy/visualization/tests/test_stretch.py

```diff
@@ -106,3 +108,29 @@ def test_clip_invalid():
 
     values = stretch([-1., 0., 0.5, 1., 1.5], clip=False)
     np.testing.assert_allclose(values, [np.nan, 0., 0.70710678, 1., 1.2247448])
+
+
+@pytest.mark.parametrize('a', [-2., -1, 1.])
+def test_invalid_powerdist_a(a):
+    with pytest.raises(ValueError):
```

Comment:
you can use ``match='message'`` to check that the message raised is correct

Reply:
Done.